packages/windows/src/RPA/Windows/__init__.py

```diff
@@ -18,14 +21,18 @@
     # current environment, instead keeping them in memory.
     # Slower, but prevents dirtying environments.
     import comtypes.client
-
     comtypes.client.gen_dir = None
 
-    import uiautomation as auto
     from uiautomation.uiautomation import Logger
 
 
-class Windows(DynamicCore):
+class DynamicCore(RobotDynamicCore, RPADynamicCore):
+    """Ensures robotframework methods get in front of the vendorized ones."""
+
+
+# NOTE(cmiN): We inherit from our `DynamicCore` just to make sure the robotframework
+#  one takes precedence in front of core's vendorized one.
+class Windows(WindowsElements, DynamicCore):
```

Comment:
Ensuring we use the original `DynamicCore` and not the venorized one from #425 . (safety reasons)